gateway-js/src/__tests__/loadSupergraphSdlFromStorage.test.ts

```diff
@@ -23,402 +30,69 @@ describe('loadSupergraphSdlFromStorage', () => {
-        username: String @join__field(graph: ACCOUNTS) @join__field(graph: REVIEWS, external: true)
-        birthDate(locale: String): String @tag(name: \\"admin\\") @tag(name: \\"dev\\") @join__field(graph: ACCOUNTS)
-        account: AccountType @join__field(graph: ACCOUNTS)
-        metadata: [UserMetadata] @join__field(graph: ACCOUNTS) @join__field(graph: INVENTORY, external: true) @join__field(graph: REVIEWS, external: true)
-        ssn: String @join__field(graph: ACCOUNTS)
-        goodDescription: Boolean @join__field(graph: INVENTORY, requires: \\"metadata { description }\\")
-        vehicle: Vehicle @join__field(graph: PRODUCT)
-        thing: Thing @join__field(graph: PRODUCT)
-        reviews: [Review] @join__field(graph: REVIEWS)
-        numberOfReviews: Int! @join__field(graph: REVIEWS)
-        goodAddress: Boolean @join__field(graph: REVIEWS, requires: \\"metadata { address }\\")
-      }
-
-      type UserMetadata
-        @join__type(graph: ACCOUNTS)
-        @join__type(graph: INVENTORY)
-        @join__type(graph: REVIEWS)
-      {
-        name: String @join__field(graph: ACCOUNTS)
-        address: String @join__field(graph: ACCOUNTS) @join__field(graph: REVIEWS, external: true)
-        description: String @join__field(graph: ACCOUNTS) @join__field(graph: INVENTORY, external: true)
-      }
-
-      type Van implements Vehicle
-        @join__implements(graph: PRODUCT, interface: \\"Vehicle\\")
-        @join__implements(graph: REVIEWS, interface: \\"Vehicle\\")
-        @join__type(graph: PRODUCT, key: \\"id\\")
-        @join__type(graph: REVIEWS, key: \\"id\\", extension: true)
-      {
-        id: String!
-        description: String @join__field(graph: PRODUCT)
-        price: String @join__field(graph: PRODUCT) @join__field(graph: REVIEWS, external: true)
-        retailPrice: String @join__field(graph: REVIEWS, requires: \\"price\\")
-      }
-
-      interface Vehicle
-        @join__type(graph: PRODUCT)
-        @join__type(graph: REVIEWS)
-      {
-        id: String! @join__field(graph: PRODUCT)
-        description: String @join__field(graph: PRODUCT)
-        price: String @join__field(graph: PRODUCT)
-        retailPrice: String @join__field(graph: REVIEWS)
-      }",
-      }
-    `);
+    expect(result).toMatchObject({
+      id: 'originalId-1234',
+      supergraphSdl: getTestingSupergraphSdl(),
+    });
   });
 
+  it('Throws error if all Uplink URLs fail', async () => {
+    mockSupergraphSdlRequest("originalId-1234").reply(500);
+    mockOutOfBandReportRequestSuccess();
+    mockSupergraphSdlRequestIfAfter("originalId-1234", mockCloudConfigUrl2).reply(500);
+    mockOutOfBandReportRequestSuccess();
+
+    const fetcher = getDefaultFetcher();
+    await expect(
+      loadSupergraphSdlFromUplinks({
+        graphRef,
+        apiKey,
+        endpoints: [mockCloudConfigUrl1, mockCloudConfigUrl2],
+        errorReportingEndpoint: mockOutOfBandReporterUrl,
```

Comment:
Again here we can disable the error reporter and avoid having to mock the requests

Reply:
I set `errorReportingEndpoint` to undefined here.